### PR TITLE
2074 Add download button for files in streampipes

### LINKED ISSUE #2074: Add download button for files in StreamPipes
### Body

StreamPipes allows the upload of files for various reasons. All uploaded files can be viewed in the module `File Management`.

image

Unfortunately, there is currently no way to download a file and inspect it, in case one wants to see what the file contains.
Therefore we should add a download action directly beneath the remove action for every file.

### Mentoring

As this issue is marked as a good first issue: one of @dominikriemer, @tenthe, or @bossenti are happy to provide help for getting started, just tag (one of) them if you want to start working on this issue and need some help.
This issue is ideal for people who are interested in getting started with StreamPipes and what to explore both worlds: the backend and the UI.

### StreamPipes Committer

I acknowledge that I am a maintainer/committer of the Apache StreamPipes project.

### PR DESCRIPTION
**Pull request by IsaakKrut**
closes #2074

image

### CODE REVIEW

File: ui/src/app/files/components/file-overview/file-overview.component.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ import { MatTableDataSource } from '@angular/material/table';
 import { MatPaginator } from '@angular/material/paginator';
 import { ConfirmDialogComponent } from '@streampipes/shared-ui';
 import { MatDialog } from '@angular/material/dialog';
+import * as FileSaver from 'file-saver';
```
Comment (tenthe) [inline ui/src/app/files/components/file-overview/file-overview.component.ts:25]: Just one minor comment. Maybe we can replace this with `import { saveAs } from 'file-saver';` 
I saw that we use the file-saver like you did it in `PipelinesComponent` as well, but I guess it is better to import it without the `*`.

**Review by tenthe — COMMENTED** [review]
Thank you, @IsaakKrut, for submitting this PR and welcome to the StreamPipes community. It looks very good! I particularly like the integration into the tests and the inclusion of a screenshot how the UI looks.

File: ui/src/app/files/components/file-overview/file-overview.component.ts (hunk 1 above)
Comment (IsaakKrut) [reply ui/src/app/files/components/file-overview/file-overview.component.ts]: Thanks! Updated the import statement

**Review by tenthe — APPROVED** [review]
Thank you very much

**Review by bossenti — APPROVED** [review]
Works like a charm 🤩 

Thank you very much @IsaakKrut

**Comment by dominikriemer** [comment]
Thanks a lot @IsaakKrut, especially for providing the tests as well.
Congrats to your first merged PR! 🥇

**Pull request merged**